src/themes.lisp

```diff
@@ -26,9 +26,25 @@ function that takes a DOCUMENT and returns NIL or a STRING for template insertio
   (or (find-package (format nil "~:@(coleslaw.theme.~A~)" name))
       (error 'theme-does-not-exist :theme name)))
 
-(defun theme-fn (name &optional (package (theme *config*)))
+(defun get-djula-theme (name)
+  (symbol-value (intern (string-upcase (format nil "*djula-~A-template*" (string name))))))
+
+(defun theme-fn (name &optional
+                        (package (theme *config*))
+                        (templating-engine *templating-engine*))
   "Find the symbol NAME inside PACKAGE which defaults to the theme package."
-  (find-symbol (princ-to-string name) (theme-package package)))
+  (case templating-engine
```

Comment:
To make this extensible, we should dispatch on hash-table instead, something like

```lisp
(if-let (theme-fn (gethash templating-engine *available-engines*))
  theme-fn
  (error "Theme-fn not found. Unknown template-engine ~A" templating-engine))
```